docs/LwM2M_Client/OpenThread/Adding_device.md

```diff
@@ -0,0 +1,145 @@
+# Adding nRF board to OpenThread
+
+Integrate your Nordic nRF52840-DK board to an active OpenThread Border Router.
+
+## Prerequisites
+
+- An active **OpenThread Border Router**.
+- The **nRF52840-DK** board with a USB cable.
+- Installed **minicom** or **RealTerm** (for Linux or Mac) or **PuTTy** (for Windows) or other serial communication program.
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+- A user with access to the [**Coiote IoT Device Management**](https://eu.iot.avsystem.cloud/ui/device/inventory) platform and appropriate permissions.
+
+## Connecting to the OpenThread
+
+Check your active Border Router **OTBR_IP_address** and **OTBR_port** on which your Border Router works. This will be necessary for further steps.
+
+### Connection through Wi-Fi
+
+You have to be connected to the same Wifi network as your OpenThread Border Router. To connect your device to Border Router through Wifi, you should go to `OTBR_IP_address`.
+
+!!! Note
+    There is an option to connect to your Border Router through ssh by `ssh -L OTBR_port:OTBR_IP_addres:80 -L 8081:OTBR_hostname` command, then go to: `http://OTBR_IP_address:80`. The additional forwarding for port 8081 is necessary for the ***Topology*** view to work. Also, port 80 is necessary due to the HTTP protocol.
+
+When you are connected you should see the OpenThread main page:
+
+![OT Main Page](images/ot_main_page.PNG "OT Main Page")
+
+### Creating an OpenThread network
+
+On the left side, select the option ***Form***, and a new page will be displayed for the network creation. Click the form and a message should pop up to let you know about the operation's success. ![Form Network](images/form_network.png "Form Network"){:style="float: left;margin-right: 1177px;margin-top: 17px; margin-bottom: 17px;"}
```

Comment:
Did you check if the default configuration from this ***Form*** page works?

Reply:
I read about this [here](https://jerome-silabs.github.io/SE_FAE_team/OpenThread/Applications/OpenThread_Border_Router/create_network.html) but I didn't try this